From e305d6bc52a7965affe17935c1fa18b828aee353 Mon Sep 17 00:00:00 2001
From: Zachary Goldberg <zgs@seas.upenn.edu>
Date: Sat, 27 Sep 2008 14:26:03 -0400
Subject: [WineHQ][WWN] Remove the google adds box
To: wine-patches <wine-patches@winehq.org>, Jeremy Newman <jnewman@codeweavers.com>

Was up to support WineConf, should be taken down now that WineConf has happened.
---
 site |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/site b/site
index 4404b13..4aefa5c 100644
--- a/site
+++ b/site
@@ -120,7 +120,7 @@ function home_page ()
                   'news_box'    => $news_box,
                   'latest_box'  => $latest_box,
                   'quote_box'   => $quote_box,
-                  'ad_box'      => $ad_box,
+//                  'ad_box'      => $ad_box,
                   'wwn_box'     => $wwn_box,
                   'sponsor_box' => $sponsor_box
                  );
-- 
1.5.6.4

